### DISCUSSION TITLE
[Suggestion] Async Version

### DISCUSSION THREAD

**Discussion by Gowixx** [ThioJoe/YT-Spammer-Purge#75] [Ideas]
Title says it all.

**Comment by TechStudent10** [comment]
If possible, yeah, an asynchronous version would be nice.

**Comment by Gowixx** [comment]
or threading it could possible work too

**Comment by Ahsoka** [comment]
An async version is possible.  An async library for all of Google's APIs is available here: https://github.com/omarryhan/aiogoogle

**Reply by TechStudent10** [reply]
nice!

**Reply by Gowixx** [reply]
Great find. I could look into it later

**Reply by Ahsoka** [reply]
I also have a quite a bit of experience with the library so if you interested in making an async rework with it, I'd be happy to help!

See here for reference: https://github.com/Ahsoka/teckla
